dressup/src/pages/Homepage/TrendingSection/TrendingSection.js

```diff
@@ -0,0 +1,46 @@
+import "./TrendingSection.css";
+import { PrimaryNavbutton } from "../../../components/Navbar";
+import img3 from "../../../images/handbags.jpeg";
+import img4 from "../../../images/watch.jpeg";
+import img5 from "../../../images/sport-shoes.webp";
```

Comment:
Suggestion: you can make index.js file for images and import all images form there